### PR TITLE
Improved standard buttons render

### PR DESCRIPTION
**Pull request by SSPkrolik**

### CODE REVIEW

File: nimx/composition.nim [hunk 1]
```diff
@@ -207,7 +207,13 @@ vec4 composeDistanceFuncDebug(float dist) {
 }
 
 void drawShape(float dist, vec4 color) {
-    gl_FragColor = mix(gl_FragColor, color, fillAlpha(dist));
+    if (gl_FragColor.w == 0.0) {
```
Comment (yglukhov) [inline nimx/composition.nim]: Could you please define another function e.g. `drawFirstShape` or smth? And then use it.

File: nimx/button.nim [hunk 2]
```diff
@@ -111,38 +111,91 @@ proc drawRegularStyle(b: Button, r: Rect) {.inline.} =
...
-        let insetVal = bezelRect.height * 0.18
-        let checkMarkRect = bezelRect.inset(insetVal, insetVal)
-        c.strokeWidth = 0
-        c.fillColor = selectionColor
-        c.drawRoundedRect(checkMarkRect, checkMarkRect.height / 2)
+        radioButtonComposition.draw bezelRect:
+            setUniform("uStrokeColor", selectionColor)
+            setUniform("uFillColor", selectionColor)
+            setUniform("uRadioValue", bezelRect.height * 0.3)
+            setUniform("uRadius", bezelRect.height / 2.0 - 1.0)
```
Comment (yglukhov) [inline nimx/button.nim]: Radius can be calculated from bounds inside the shader.

File: nimx/composition.nim (hunk 1 above)
Comment (SSPkrolik) [inline nimx/composition.nim]: @yglukhov fixed

File: nimx/button.nim (hunk 2 above)
Comment (SSPkrolik) [inline nimx/button.nim]: @yglukhov fixed

**Comment by SSPkrolik** [comment]
@yglukhov Fixed all that was commented

**Pull request merged**